docs/guide/clustering_aws_s3.md

```diff
@@ -24,6 +24,10 @@ See <http://docs.aws.amazon.com/AmazonS3/latest/dev/s3-access-control.html>.
 The secret key cannot be retrieved again after the key has been created, so don't lose it.
 (However, you can create new keys if needed.)
 
+!!! note
+
+    Make sure that your bucket is [configured as Public](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/block-public-access-bucket.html) to avoid facing 403 errors, as the current S3 handler requires files to be stored publicly for linking to work as intended.
```

Comment:
@DominikaK updated text here

note: "linking" is kind of missleading on second looki t could be rewritten as
```suggestion
    Make sure that your bucket is [configured as Public](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/block-public-access-bucket.html) to avoid facing 403 errors, as the current S3 handler is meant to store files publicly so they can be served directly from S3.
```